docs/reports/markdownified-datasets.md

```diff
@@ -0,0 +1,64 @@
+# Dataset Card: Markdownified Datasets
+
+This dataset collection consists of several large-scale text corpora that have been processed and tokenized through a custom fork of Resiliparse. The datasets are primarily intended for language model training and research purposes.
+
+## Dataset Statistics
+
+| Dataset | Token Count | Approximate Size | Source |
+|---------|------------|------------------|--------|
+| Wiki | 8587224558 | 8.59B tokens | [Wikipedia Dump](https://dumps.wikimedia.org/other/enterprise_html/runs/20241201/enwiki-NS0-20241201-ENTERPRISE-HTML.json.tar.gz) |
+| Arxiv No Problem | 2742463924 | 2.74B tokens | [ArXiv Dump](https://sigmathling.kwarc.info/resources/ar5iv-dataset-2024/) |
+| Arxiv Warning | 19552307274 | 19.6B tokens | [ArXiv Dump](https://sigmathling.kwarc.info/resources/ar5iv-dataset-2024/) |
+| Stack Exchange | TODO | TODO | [Stack Exchange Dump] |
+
+## Processing Methodology
+
+These datasets were processed using our custom fork of Resiliparse which simplified the raw HTML DOM, the simplified DOM was then processed by our custom implementation of Markdownify. The exact modifications and enhancements made to the original Resiliparse are documented in the next section, the processing pipeline appears to have:
```

Comment:
link to fork of resiliparse, original version of resiliparse, our modified markdownify, original markdownify